src/openzaak/components/catalogi/tests/admin/test_resultaattype_admin.py

```diff
@@ -589,3 +589,34 @@ def test_create_resultaattype_for_published_zaaktype_not_allowed(self, m):
             },
         )
         self.assertEqual(ResultaatType.objects.count(), 0)
+
+    def test_resultaattype_detail_with_read_only_user_invalid_domain(self, m):
+        user = UserFactory.create(is_staff=True)
+        view_resultaattype = Permission.objects.get(codename="view_resultaattype")
+        user.user_permissions.add(view_resultaattype)
+        self.app.set_user(user)
+
+        selectielijst_api = "https://selectielijst.openzaak.nl/api/v1/"
+        procestype_url = (
+            f"{selectielijst_api}procestypen/e1b73b12-b2f6-4c4e-8929-94f84dd2a57d"
+        )
+        omschrijving_url = (
+            "https://selectielijst.openzaak.nl/api/v1/"
+            "resultaattypeomschrijvingen/e6a0c939-3404-45b0-88e3-76c94fb80ea7"
+        )
+        mock_selectielijst_oas_get(m)
+        mock_resource_get(m, "procestypen", procestype_url)
+        mock_resource_get(m, "resultaattypeomschrijvingen", omschrijving_url)
+        resultaattype = ResultaatTypeFactory.create(
+            zaaktype__selectielijst_procestype=procestype_url,
+            selectielijstklasse="http://some-domain.local",
+            resultaattypeomschrijving=omschrijving_url,
+        )
+        url = reverse("admin:catalogi_resultaattype_change", args=(resultaattype.pk,))
+
+        response = self.app.get(url)
+
+        self.assertEqual(response.status_code, 200)
+
+        expected_error = "De selectielijstklasse is niet afkomstig uit de Referentielijsten API-service"
```

Comment:
Once the string is made translatable, we also have to translate it in tests to avoid errors
```suggestion
        from django.utils.translations import gettext as _
        ...
        expected_error = _("De selectielijstklasse is niet afkomstig uit de Selectielijst API-service")
```